### ISSUE TITLE
ppo_nn_agent.gin hyperparam tuning

### ISSUE THREAD

**Issue by amirjamez** [google/ml-compiler-opt#17]
Hi @yundiqian. I was skimming through the hyperparams of https://github.com/google/ml-compiler-opt/blob/main/compiler_opt/rl/inlining/gin_configs/ppo_nn_agent.gin and it seems counterintuitive to me that both `PPOAgent.normalize_rewards` and `PPOAgent.normalize_observations` are assigned as `False`. Did that have specific reason? 

Looking at the TF docs (https://github.com/tensorflow/agents/blob/master/tf_agents/agents/ppo/ppo_agent.py#L206)), it is advised to normalize rewards and observation, so I was wondering if you had experienced with this before? 

Thanks!
- Amir
